assets/pages/contributions-thankyou/oneoffContributionsThankyou.jsx

```diff
@@ -49,7 +48,7 @@ const content = (
             componentType: 'ACQUISITIONS_OTHER',
             id: 'oneoff-thankyou-page-recurring-upsell',
             products: ['RECURRING_CONTRIBUTION'],
-            upSellCampaignCode,
+            campaignCode: upSellCampaignCode,
```

Comment:
I think the `campaignCode` and `id` should be the same here...they look like they might actually be, but if so then we should define them in the same way

Reply:
👍